templates/base_index.html

```diff
@@ -1382,8 +1382,15 @@ <h2>
           var takeovers = JSON.parse(xhr.responseText);
 
           // Get the selected takeovers based on the primary language
+          var currentUserCountry = "{{ current_user_country }}"
```

Comment:
I'm assuming this is in the homepage source? The homepage is fairly heavily cached, which means everyone will appear to have the country of whichever user most recently regenerated the cache. Obviously this isn't ideal.

If we're going to pass the country through from the backend then it has to be done on an endpoint that is uniquely generated for each user.So I guess your options are to:

a) do the country checking inside the `/takeovers.json` endpoint, and make sure that endpoint is private to each user (I don't know if this has negative performance implications)
b) create a separate endpoint just for checking country (`/user-country.json`) and pull in the country from here in the JS. This would mean that `/takeovers.json` would be more cacheable - probably a good idea since that endpoint needs to query discourse.
c) instead of writing our own `/user-country.json` endpoint, we could use a geoip service to get the country client-side. I don't know if any of these services are free or reliable enough for us to use them instead. https://www.maxmind.com/en/home, https://ipregistry.co/, https://www.ip2location.com/, https://ipinfo.io/

Reply:
Isn't maxmind what we are using currently?

I def prefer option b, but I'll think about it and do an implementation see how it works.

Reply:
> Isn't maxmind what we are using currently?

Yes I think that's right, but I think what we're using is a download of the maxmind database rather than using their API. But I might be wrong.